lightningd/plugin.c

```diff
@@ -1821,7 +1821,7 @@ plugin_populate_init_request(struct plugin *plugin, struct jsonrpc_request *req)
 	if (ld->proxyaddr) {
 		json_add_address(req->stream, "proxy", ld->proxyaddr);
 		json_add_bool(req->stream, "torv3-enabled", ld->config.use_v3_autotor);
-		json_add_bool(req->stream, "use_proxy_always", ld->use_proxy_always);
```

Comment:
Needs to go through a deprecation cycle...

Reply:
k, I added `if (deprecated_apis) json_add_bool(req->stream, "use_proxy_always" ...` . Is that sufficient?

Reply:
Weird, because they *should* be!

Yes, that's sufficient